src/main/java/seedu/address/logic/commands/DeleteAliasCommand.java

```diff
@@ -0,0 +1,58 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.logic.parser.ValidCommand;
+import seedu.address.logic.parser.exceptions.AliasNotFoundException;
+import seedu.address.model.Model;
+
+public class DeleteAliasCommand extends Command {
+
+    public static final String COMMAND_WORD = "release";
```

Comment:
Maybe we should use command `deletealias` and `deletea`?

Reply:
I updated in my new commit